api/converter_utils.go

```diff
@@ -513,15 +513,17 @@ func signedTxnWithAdToTransaction(stxn *sdk.SignedTxnWithAD, extra rowData) (gen
 	case sdk.HeartbeatTx:
 		hb := stxn.Txn.HeartbeatTxnFields
 		hbTxn := generated.TransactionHeartbeat{
-			HbAddress: hb.HbAddress.String(),
+			HbAddress:     hb.HbAddress.String(),
+			HbKeyDilution: hb.HbKeyDilution,
```

Comment:
Is it convention here to alphabetize? In go-algorand, I grouped Seed, KeyDilution, and VoteID together because they are similar - "extra" stuff that must be present for the early crypto check, but almost seems like it doesn't belong.

Reply:
This is actually due to how the OAS 3 generator works (it sorts the fields alphabetically). The `HeartbeatTxnFields` type in `go-algorand-sdk` has the ordering matching `go-algorand`.

Reply:
ok by me